examples/getuser.php

```diff
@@ -0,0 +1,68 @@
+<?php
+require_once('../Payone.php');
+require_once('config.php');
+
+
+/**
+ * This request is used to retrieve existing user data on the BSPAYONE platform.
+ * The primary key for user data is the parameter 'userid', which can be obtained from the response of the preauthorization or authorization request
+ * Usecases:
+ * - use this request to receive structured JSON containing personal data
+ * - you can also retrieve usertokens to migrate user data between merchant accounts
+ * PLEASE NOTE: Like payment requests, we will charge a fee for each of these requests. So relying on these will get expensive very fast.
+ */
+$parameters = [
+    "request" => "getuser",
+    "userid" => '123456789', // unique userid
+    "type" => 'userdata', // retrieves user data as structured JSON
+    // "customerid" => "12345", // you can also use your own customer identifier if you specified it in the initial payment request
+    // "getusertoken" => "yes", // returns a token for migration purposes. You can then use a payment request (preauthorization or authorization) on another mid to migrate the user data
+];
+
+$request = array_merge($defaults, $parameters, $personalData);
+ksort($request);
+print_r($request);
+$response = Payone::sendRequest($request);
```

Comment:
Doesn't the `"type" => 'userdata'` directive require to set the `Accept: application/json` header in the HTTP communication? The `Payone` class should be patched to allow setting the header then.